Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 default-members = ["apollo-router", "apollo-router-core"]
 members = [
+    "apollo-relay",
```

Comment:
I wouldn't recommend naming a component "relay" if there are alternatives (especially if it's something that might be externally visible, which I think everything in the router will be)? Relay is the name of the second-most-popular JS GraphQL client (the original one from Facebook) so I think it'll be confusing if we use that name for a component.

Reply:
I only just changed it to "apollo-relay" from "usage-agent" because I thought that was a better name. I'm not attached to it, but it does represent what it does (relay usage data to our ingress) really well. I'll see if I can dream up something better.

Reply:
Router team discussion today leant towards embracing the space theme. The proposed rename was from "relay" to "spaceport".
I'm making that change now.